vgrep.go

```diff
@@ -564,6 +567,14 @@ func (v *vgrep) dispatchCommand(input string) bool {
 		return v.commandRefine(cmdArray[1])
 	}
 
+	if cmdArray[0] == "n" || cmdArray[0] == "newsearch" {
+		if len(cmdArray) < 2 {
+			fmt.Println("newsearch expects at least a pattern")
+			return false
+		}
+		return v.commandNewSearch(cmdArray[1])
```

Comment:
I'd always want to see the new locations/matches after each search and think we could print here directly. What do you think?

Reply:
Agreed, I'll do that. Thanks for the review!